orientdb/src/test/java/com/yahoo/ycsb/db/OrientDBClientTest.java

```diff
@@ -227,10 +259,11 @@ public void scanTest() {
     int testIndex = startIndex;
 
     // Check each vector row to make sure we have the correct fields
-    for (HashMap<String, ByteIterator> result: resultVector) {
+    for (HashMap<String, ByteIterator> result : resultVector) {
```

Comment:
The comment a few lines above this about index starting at 1 can be removed. Change line 243 to startIndex = 0.